FoundationsMastery/CreateFactorial.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FoundationsMastery.models;
+
+namespace FoundationsMastery
+{
+    class CreateFactorial : Factorial
+    {
+        public override int Iterative(int n)
```

Comment:
1. The assignment called for you to create **2 additional classes** not related to the `Factorial` or `CustomString` implementations.
2. This doesn't really demonstrate how to use `override` given this implementation is that same as what's in your `Factorial` class.